src/main/java/com/lotto/model/LottoService.java

```diff
@@ -0,0 +1,69 @@
+package com.lotto.model;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoService {
+	private static final int TICKET_PRICE = 1000;
+
+	public LottoTickets publishLottoTickets(int purchaseAmount) {
+		int ticketCount = calculateAmount(purchaseAmount);
+		return LottoTickets.createLottoTickets(ticketCount);
+	}
+
+	public Map<Rank, Integer> checkResult(LottoTickets tickets, List<Integer> winningNumbers, int bonusNumber) {
+		List<Lotto> ticketList = tickets.getTickets();
+
+		Map<Rank, Integer> result = new LinkedHashMap<>() {{
+			put(Rank.FIFTH, 0);
+			put(Rank.FOURTH, 0);
+			put(Rank.THIRD, 0);
+			put(Rank.SECOND, 0);
+			put(Rank.FIRST, 0);
+		}};
+
+		for (Lotto lotto : ticketList) {
+			int ticketResult = checkTicket(lotto.getTicket(), winningNumbers);
```

Comment:
더 명확한 이름을 써 주면 좋겠네요. 예를 들어
```suggestion
			int matchCount = countMatchNumbers(lotto.getTicket(), winningNumbers);
```